conda-env-dev.yml

```diff
@@ -1,10 +1,11 @@
 name: primo
 channels:
   - conda-forge
-  - defaults
+  - gurobi
 dependencies:
   - python >=3.9
   - pip
   - scip >=9.1.0
+  - gurobi
```

Comment:
Is there a reason for installing gurobi in this manner instead of `pip install gurobipy`?

Reply:
For some unknown reason, the Binder environment (mybinder.org) is unable to find gurobi via `pip install gurobipy` when working on Anaconda.

Decided to follow the officially recommended method by Gurobi for the case of Anaconda (https://support.gurobi.com/hc/en-us/articles/12872889819921-How-do-I-use-conda-to-install-Gurobi-in-Anaconda)